routes/users.js

```diff
@@ -34,6 +34,7 @@ router.patch(
   userValidator.validateUpdateRoles,
   users.updateRoles
 );
+router.patch("/archived", authenticate, authorizeRoles([SUPERUSER]), users.archiveUserIfNotInDiscord);
```

Comment:
archived is not a resource, Please change the API name. The API names must represent resource